theano/tensor/opt.py

```diff
@@ -3936,6 +4029,110 @@ def local_elemwise_sub_zeros(node):
         return [T.zeros_like(node.inputs[0])]
 
 
+@register_specialize
+@register_stabilize
+@register_canonicalize
+@gof.local_optimizer([T.Elemwise])
+def local_useless_elemwise_comparison(node):
+    """...
+
+    :note: Those case appear in the graph generated around scan.  This
+           don't remove much computation, but make the graph easier to
+           read.
```

Comment:
Maybe this could be written more clearly. Also there are some grammar mistakes. Something like:

These cases appear in the graph generated by scan. This will not reduce computation but will make the graph easier to read.